helpers/constants.ts

```diff
@@ -71,6 +71,7 @@ export const chainIdInfoMap: Map<number, ChainIdInfo> = new Map()
   }); // linea-sepolia
 
 export const CONTRACT_NAMES = Object.freeze({
```

Comment:
Would it be better to unite `CONTRACT_NAMES` and `UNIFIED_CONTRACT_ADDRESSES` and `create2AddressesInfo` into one data structure?

Reply:
Yes, now that we have different contract information in constants we could create one data structure. I'll review it.